bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md

```diff
@@ -0,0 +1,212 @@
+# Recursive Bayseian Filters for Linear Gaussian Systems
+
+### Scope
+
+The recursive Bayesian filter describes how the probability distribution of a random signal varies with time.
+Astoundingly, if the signal is the output of a linear system with a Gaussian state distribution, the recursive Bayesian filter exactly reproduces the Kalman filter.
+
+### Objective
+
+Derive the Kalman filter from the recursive Bayesian filter subject to linear dynamics and Gaussian distributions, and comment on the connection between the two filters.
+
+## Introduction
+
+The recursive Bayesian filter (RBF) is the most general possible statement of how one's belief about the value of a random signal changes over time.
+If one wishes to estimate the state of an uncertain system, the RBF exactly describes how the probability of each state varies over time.
+This is the math that underlies any state estimation problem, from localizing the position of an aircraft using GPS to determining the orientation of a satellite using a star tracker.
+
+However, the RBF is challenging to use in practice because of the difficulty of computing the normalization factor in Bayes' rule, upon which the RBF is based, for systems with many possible states.
+Due to this difficulty in computation, it is useful to make some simplifying assumptions in order to leverage the power of the RBF.
+In particular, we will assume that
+
+1. the signal is the output of a linear system, and
+2. the state of the system follows a Gaussian distribution.
+
+These are handy assumptions because a Gaussian distribution subject to linear dynamics remains Gaussian.
+
+## Preliminaries
+
+### Gaussian random variables
+
+For a random variable $X$, a particular value or sample is represented by the lowercase $x \in \mathbb{R}^n$.
+When we say $X \sim \mathcal{N}(\mu, \Sigma)$, we mean that $X$ is "Gaussian": it follows a Gaussian or normal distribution with mean $\mu \in \mathbb{R}^n$ and covariance $\Sigma \in \mathbb{R}^{n \times n}$:
+$$
+\begin{equation}
```

Comment:
Equations are not rendering properly. Needs to be corrected.

Reply:
Thanks for catching this! Turns out I just needed to add extra newlines above and below the equation blocks.